packages/tools/devtools/devtools/api-report/devtools.api.md

```diff
@@ -39,4 +39,6 @@ export { IDevtoolsLogger }
 // @alpha
 export function initializeDevtools(props: DevtoolsProps): IDevtools;
 
+// (No @packageDocumentation comment for this package)
```

Comment:
Not against this but I'm surprised to see it in this PR...?

Reply:
I agree it's odd. CI failed with an extraneous edit without it.